src/state-summary/state-card-content.html

```diff
@@ -35,7 +35,28 @@
     'inputChanged(hass, inDialog, stateObj)',
   ],
 
-  _inputChanged: function (hass, inDialog, stateObj, stateType) {
+  _isLoaded: function (name) {
+    var elem = document.createElement(name);
```

Comment:
There is no need to do this, HTML imports will only be loaded once. `importHref` has a built-in caching mechanism (as per the spec that it polyfills).